scipy/optimize/_basinhopping.py

```diff
@@ -15,22 +15,21 @@ class Storage(object):
     """
     Class used to store the lowest energy structure
     """
-    def __init__(self, x, f):
-        self._add(x, f)
+    def __init__(self, minres):
+        self._add(minres)
 
-    def _add(self, x, f):
-        self.x = np.copy(x)
-        self.f = f
+    def _add(self, minres):
+        self.minres = minres
```

Comment:
If you wanted to be extra safe here you could do 

    self.minres.x = np.copy(minres.x)

I seem to recall that np.copy() was important here.  Possibly because for 1D minimization the minimizer might return a simple float rather than an array of length 1.